ui/raidboss/data/timelines/o6n.txt

```diff
@@ -0,0 +1,68 @@
+# Omega - Sigmascape V2.0 - O6N
+
+hideall "--Reset--"
+hideall "--sync--"
+
+0 "--Reset--" sync /(Removing combatant Demon Chadarnook|has initiated a ready check)/ window 10000 jump 0
```

Comment:
What's with the "has initiated a ready check" log here? Do you find that the timelines aren't stopping when out of combat?

Reply:
Nah, just habit from when I made them for the old ACT_Timeline without the combat detection stuff. The start block I mostly just copy/paste from old ones.